src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/AsnEncodedDataCollection.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    public sealed class AsnEncodedDataCollection : ICollection
+    {
+        public AsnEncodedDataCollection()
+        {
+            _list = new LowLevelListWithIList<AsnEncodedData>();
```

Comment:
>  I assumed there was an unspoken reason we didn’t want to take the dependency in this package

Unless I'm missing something, we already have one: System.Collections is in the project.json.